src/lib/booleans.js

```diff
@@ -0,0 +1,91 @@
+const { arrayToCSVString } = require('./arrays');
```

Comment:
It would be good to see these completed and added into the test suites. In the meantime, I would add a comment about your intention to complete these in the future to the README